packages/smart-wallet/src/invitations.js

```diff
@@ -166,3 +166,5 @@ export const makeInvitationsHelper = (
   return invitationFromSpec;
 };
 harden(makeInvitationsHelper);
+
+/** @typedef {ReturnType<makeInvitationsHelper>} InvitationFromSpec */
```

Comment:
this type is just `ERef<Invitation>`. it shouldn't matter where an invitation comes from.
```suggestion
```